pkg/core/message.go

```diff
@@ -86,6 +86,11 @@ type MessageHeader struct {
 	DataHash  *fftypes.Bytes32      `ffstruct:"MessageHeader" json:"datahash,omitempty" ffexcludeinput:"true"`
 }
 
+type MessageTransactions struct {
+	Batch   *fftypes.UUID `ffstruct:"MessageTransactions" json:"batch,omitempty" ffexcludeinput:"true"`
+	Related *fftypes.UUID `ffstruct:"MessageTransactions" json:"related,omitempty" ffexcludeinput:"true"`
```

Comment:
I'm not in love with the naming here, but I did find that I had to use two distinct fields rather than a simple list of IDs (because we bulk-update the messages when the batch and batch transaction are assigned).

Reply:
Also worth noting, we do not _have_ to store both. We could opt for one or the other, as the paths that compute them are totally independent - just wanted to bottom out what it would look like in each case.